### PR TITLE
fix: relax prettier version specification

### PR DESCRIPTION
**Pull request by tancnle**
# Description

This PR allows minor and patch version bump of `prettier` package.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (update or new)

# How Has This Been Tested?

Run `npm install` locally to verify that the `prettier` version is set correctly (i.e. the latest is [`2.6.2`](https://www.npmjs.com/package/prettier/v/2.6.2))

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [-] I have made corresponding changes to the documentation - not required
- [x] My changes generate no new warnings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "format:check": "prettier --check \"**/*.md\""
   },
   "devDependencies": {
-    "prettier": "2.6.1"
+    "prettier": "^2.6.2"
```
Comment (tancnle) [inline package.json:10]: This ensures the minimum supported version of prettier is `2.6.2`.

File: package-lock.json [hunk 2]
```diff
@@ -1,13 +1,12 @@
 {
   "name": "handbook",
-  "version": "0.0.1",
```
Comment (tancnle) [inline package-lock.json:3]: The `handbook` is not a NPM package so we do not need to track version.

**Review by ghost — APPROVED** [review]
Looks good!
Hopefully I don't break it again

**Comment by tancnle** [comment]
Thank you for a speedy approval @talzid 🙇🏼

**Pull request merged**